deploy/eck-stack/charts/eck-enterprise-search/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: eck-enterprise-search
+description: Elastic Enterprise Search managed by the ECK operator
+kubeVersion: ">= 1.21.0-0"
+type: application
+version: 0.11.0-SNAPSHOT
+sources:
+  - https://github.com/elastic/cloud-on-k8s
+  - https://github.com/elastic/ent-search
```

Comment:
Private repo. Let's leave that out.

Reply:
I honestly didn't know/see that. removed.